serviceconnector/src/validateServiceConnector/ValidateLinkerStep.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { IPickLinkerContext } from "../deleteServiceConnector/IPickLinkerContext";
+
+export class ValidateLinkerStep extends AzureWizardExecuteStep<IPickLinkerContext> {
+    public priority: number = 200;
```

Comment:
Add comment to explain why this is set to 200. What is it meant to come before or after?

Reply:
In all cases (create, delete, and validate) only one execute step happens, so 200 was mostly an arbitrary number. I'll set it to a smaller number in case we add more execute steps in the future.